packages/flutter/test/cupertino/scrollbar_test.dart

```diff
@@ -369,6 +370,11 @@ void main() {
     'When isAlwaysShown is true, '
     'must pass a controller or find PrimaryScrollController that is attached to a scroll view',
     (WidgetTester tester) async {
+      final FlutterExceptionHandler? handler = FlutterError.onError;
```

Comment:
@Piinks Since now the first scroll metrics initialize will call fadeoutanimationcontroller.forward(), the error may throw multiple times

Reply:
We probably won't need the first empty scroll event request now.